pkg/execution/svccatadmission/internaldns.go

```diff
@@ -55,7 +55,7 @@ func InternalDNSAdmitFunc(ctx context.Context, microsServerClient microsServerCl
 				},
 			}, nil
 		}
-		var internalDNSSpec apiinternaldns.Spec
+		var internalDNSSpec apiplatformdns.Spec
```

Comment:
This should probably be:
var platformDNSSpec apiplatformdns.Spec

Reply:
These files belong to the webhook, I have a separate ticket/PR for them